### DISCUSSION TITLE
AggregatedSummary not supported on GCP Monitoring

### DISCUSSION THREAD

**Discussion by trickster** [vectordotdev/vector#9352] [Q&A] [answered]
I am trying to prometheus_scrape -> GCP monitoring.

Prometheus source is directly parsed to Summary metric, but the field is called AggregatedSummary and this metric is not recognized by GCP monitoring

```
app_request_duration_seconds_sum xxx
app_request_duration_seconds_count yyy
```

```
vector::sinks::gcp::stackdriver_metrics: Unsupported metric type: AggregatedSummary { quantiles: [], count: yyy, sum: xxx }
```

**Comment by jszwedko** [comment, marked as answer]
Hey @trickster ! This does seem like a gap. I opened https://github.com/vectordotdev/vector/issues/9530 to track it.
